terraform/version_constraint.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  required_version = ">= 0.8.5"
```

Comment:
TBH I'm not sure what's the binary compatibility of the compiled pingdom provider. I've got local terraform 0.8.7 and the linting tests pass but can we guarantee this ?
Shall we be more strict here?